### DISCUSSION TITLE
采用自制字典（A-Z，0-9)每行一开始四个字母只能识别出一个字母

### DISCUSSION THREAD

**Discussion by hustvl-luckybear** [PaddlePaddle/PaddleOCR#12488] [Q&A]
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/62426818/fed29add-d14f-49c0-a13f-bf5cb99eacee)
上图识别的结果如下
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/62426818/a6e874f5-3155-4ec0-a112-e404c8f666dc)

**Comment by 0-Maxwei-0** [comment]
> ![image](https://user-images.githubusercontent.com/62426818/267681832-fed29add-d14f-49c0-a13f-bf5cb99eacee.png) 上图识别的结果如下 ![image](https://user-images.githubusercontent.com/62426818/267682370-a6e874f5-3155-4ec0-a112-e404c8f666dc.png)

你好！请问下这个自制字典是怎么实现的？
